dataland-document-manager/src/main/kotlin/org/dataland/documentmanager/services/conversion/PptToPdfConverter.kt

```diff
@@ -0,0 +1,43 @@
+package org.dataland.documentmanager.services.conversion
+
+import org.apache.poi.hslf.usermodel.HSLFSlideShow
+import org.dataland.datalandbackendutils.exceptions.InvalidInputApiException
+import org.jodconverter.core.document.DefaultDocumentFormatRegistry
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Component
+import org.springframework.web.multipart.MultipartFile
+
+/**
+ * Converts a ppt file to a pdf file
+ */
+@Component
+class PptToPdfConverter(
```

Comment:
why does the converter from ppt to pdf only has a validator?

Reply:
Because the baseclass already implements the converter. It is almost the same for all microsoft office documents.